sdk/include/opentelemetry/sdk/trace/recordable.h

```diff
@@ -83,7 +83,7 @@ class Recordable
    * @param attributes the attributes associated with the link
    */
   virtual void AddLink(opentelemetry::trace::SpanContext span_context,
-                       const trace_api::KeyValueIterable &attributes) noexcept = 0;
+                       const opentelemetry::common::KeyValueIterable &attributes) noexcept = 0;
```

Comment:
Same here: could we get rid of the `trace_api` definition?

Reply:
Create generic short pattern for `opentelemetry::common` as `common_api` or even as `api`?

Reply:
Sure thing - should I leave `trace_api` in as well then?

Reply:
Should this be part of the scope resolution usage PR discussed [here](https://github.com/open-telemetry/opentelemetry-cpp/pull/363#discussion_r505600440)?

Reply:
@ThomsonTan Could this be addressed as part of issue [#368 ](https://github.com/open-telemetry/opentelemetry-cpp/issues/368)?